pytorch/training/docker/1.11/py3/cu115/Dockerfile.gpu

```diff
@@ -232,15 +230,9 @@ ARG PT_TORCHAUDIO_URL
 ARG PT_TORCHDATA_URL
 ARG PT_S3_WHL_GPU
 
-# Re-install AWS-PyTorch binaries
-RUN pip uninstall -y torch \
- && pip install --no-cache-dir -U ${PT_E3_TRAINING_URL} \
- && pip uninstall -y torchvision \
- && pip install --no-deps --no-cache-dir -U ${PT_TORCHVISION_URL} \
- && pip uninstall -y torchaudio \
- && pip install --no-deps --no-cache-dir -U ${PT_TORCHAUDIO_URL} \
- && pip uninstall -y torchdata \
- && pip install --no-deps --no-cache-dir -U ${PT_TORCHDATA_URL}
+# Install AWS-PyTorch and other torch packages
```

Comment:
Removing and installing packages in such a way contributes to decrease in size?

Reply:
the main source of decrease comes from moving `torchnet` to this step.
Collapsing the commands makes it better readability.